1-docker/README.md

```diff
@@ -2,81 +2,26 @@
 
 ### Summary
 
-In this section you will learn about :
+In this section you will learn about:
 * Running Docker locally
 * Basics of Docker
 * Containerizing a simple application
 * Building and Pushing an Image
 
-### Setup Docker locally
 
-The official documentation from *docs.docker.com* is very well detailed for setting up Docker locally depending on your OS :
-* [Mac](https://docs.docker.com/docker-for-mac/)
-* [Linux](https://docs.docker.com/engine/installation/linux/) 
-* [Windows](https://docs.docker.com/docker-for-windows/)
 
-### Basics of Docker
+### Basics of Docker and Containers
 
-You can see all the commands running `docker --help` from your terminal or powershell.
+Docker has a very well structured six-part tutorial. 
+While for this workshop you don't need to go through all of them, part 1 and 2 are required:
+* [Get Started, Part 1: Orientation and setup](https://docs.docker.com/get-started)
+* [Get Started, Part 2: Containers](https://docs.docker.com/get-started/part2/)
 
-#### Base image and registry
+By the end of Part 2, you should have a simple container up and running, and understand the basic concepts of a container.
 
-All containers contain **layers**. Those layers are customizations, such as tools installations, scripts and more. 
+#### Additional Important Docker Command
```

Comment:
Since we are refering to the official docs before we can also remove this section no ?
We can just paste the cheat sheet as reminder from the [end of the part 2](https://docs.docker.com/get-started/part2/#recap-and-cheat-sheet-optional) though  : 
```
docker build -t friendlyname .  # Create image using this directory's Dockerfile
docker run -p 4000:80 friendlyname  # Run "friendlyname" mapping port 4000 to 80
docker run -d -p 4000:80 friendlyname         # Same thing, but in detached mode
docker container ls                                # List all running containers
docker container ls -a             # List all containers, even those not running
docker container stop <hash>           # Gracefully stop the specified container
docker container kill <hash>         # Force shutdown of the specified container
docker container rm <hash>        # Remove specified container from this machine
docker container rm $(docker container ls -a -q)         # Remove all containers
docker image ls -a                             # List all images on this machine
docker image rm <image id>            # Remove specified image from this machine
docker image rm $(docker image ls -a -q)   # Remove all images from this machine
docker login             # Log in this CLI session using your Docker credentials
docker tag <image> username/repository:tag  # Tag <image> for upload to registry
docker push username/repository:tag            # Upload tagged image to registry
docker run username/repository:tag                   # Run image from a registry
```

Reply:
The cheat sheet doesn't cover `logs` `ps` though.

Reply:
Good point